sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/QuickPulseSDKClientAPIsRestClient.cs

```diff
@@ -36,6 +37,9 @@ public QuickPulseResponse CustomPing(string ikey, string apikey = null, int? xMs
                 throw new ArgumentNullException(nameof(ikey));
             }
 
+            // Prevents the http operations from being instrumented.
+            using var scope = SuppressInstrumentationScope.Begin();
```

Comment:
Why would this happen here during each call instead of set once on the dedicated thread?

Reply:
hmmm. I like that better. 
I initially thought to suppress the smallest scope possible, but I was concerned about the allocations.
Your suggestion should be much better, i'll make the change.